### PR TITLE
Adds a page layout without a TOC

### PR DESCRIPTION
**Pull request by dialpad-drew**
## Description
This PR adds a page layout without a TOC. It also cleans up the doc site grid's `[content]` width so the pages are more consistently laid out. Previously, `min-content` could make some of the layout proportions different from page-to-page.

This PR also hides the side navigation on `md` breakpoints for the time-being. This way we can test the main content's and documentation's responsive.

**Note:** I don't really like the layout naming and would prefer a little more intention behind the name, but that's not where we are at the moment.

### CODE REVIEW

File: docs/assets/css/dialtone-docs.css [hunk 1]
```diff
@@ -263,11 +263,20 @@ pre[class*="language-"].line-numbers > code {
   grid-area: toc !important;
 }
 .d-gl-docsite {
+  grid-column-gap: 4.8rem;
+          grid-column-gap: 4.8rem;
```
Comment (ryandialpad) [inline docs/assets/css/dialtone-docs.css:267]: Looks like this is duplicated?

Comment (ryandialpad) [reply docs/assets/css/dialtone-docs.css:267]: Looking below, it looks intentional. Do you know the reason for the duplication?

**Review by ryandialpad — APPROVED** [review]
w/ comment

Great Work!

File: docs/assets/css/dialtone-docs.css (hunk 1 above)
Comment (dialpad-drew) [reply docs/assets/css/dialtone-docs.css:267]: Hmmm, I don't. This is a generated filed. Let me re-build and see if it persists.

Comment (dialpad-drew) [reply docs/assets/css/dialtone-docs.css:267]: I see it locally after re-builds.I think it's in the LESS-to-CSS compilation and may only be related to `column-gap`. I can replicate it on other selectors. I'll open an issue.

```
.toc li > ol {
  margin-left: var(--su16);
  column-gap: 4.8rem;
}
```
...compiles to...
```
.toc li > ol {
  grid-column-gap: 4.8rem;
        grid-column-gap: 4.8rem;
  margin-left: var(--su16);
  -webkit-column-gap: 4.8rem;
           -moz-column-gap: 4.8rem;
                column-gap: 4.8rem;
}
```

**Pull request merged**